docs/resources/service.md

```diff
@@ -48,3 +48,11 @@ In addition to all arguments above, the following attributes are exported:
 
 * `key` - The service's API key.
 * `slug` - The slug identifies the service in URIs or where a username is required.
+
+## Import
+
+This resource can be imported using the organization slug, and the service slug:
+
+```shell
+terraform import cloudsmith_service.my_service my-organization.my-service
+```
```

Comment:
Should we call out the API key retrieval limitations for service accounts here as well?